cs/quick-start/kapitola-3.md

```diff
@@ -112,18 +112,18 @@ Následující entita bude reprezentovat autora (připomeňme si, že autoři mo
 namespace Model\Entity;
 
 /**
-* @property Book[] $books m:belongsToMany
-* @property Book[] $reviewedBooks m:belongsToMany(reviewer_id)
-* @property string|null $web
-*/
+ * @property Book[] $books m:belongsToMany
+ * @property Book[] $reviewedBooks m:belongsToMany(reviewer_id)
+ * @property string|null $web
+ */
 class Author extends Person
 {
 
 	public function getReferencingTags()
 	{
 		$tags = array();
 		foreach (array(null, 'reviewer_id') as $viaColumn) {
-			foreach ($this->row->referencing('book', null, $viaColumn) as $book) {
+			foreach ($this->row->referencing('book', $viaColumn) as $book) {
```

Comment:
Zkoušel ([viz](https://github.com/LeanMapper/quickstart/blob/master/app/Model/Entity/Author.php#L17)), dává to stejné [výsledky](https://github.com/LeanMapper/quickstart/blob/master/output.txt#L79-L105) jako původní kód. Víc jsem to ale neřešil.